coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -186,13 +238,41 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
         metadata = cls.Metadata(*(str(docstyle_settings.get(req_setting, ''))
                                   for req_setting in metadata_settings))
 
+        try:
+            docstyle_settings['class_padding']
+            class_padding = cls.ClassPadding(
+                *(int(padding) for padding in tuple(
+                    docstyle_settings.get('class_padding', ''))))
+        except IndexError:
+            class_padding = cls.ClassPadding('', '')
+
+        try:
+            docstyle_settings['function_padding']
+            function_padding = cls.FunctionPadding(
+                *(int(padding) for padding in tuple(
+                    docstyle_settings.get('function_padding', ''))))
```

Comment:
when this reads `function_padding` from `.coalang` the values are just read as a strings.
suppose `function_padding = 1, 0`
This will be read as 
`docstyle_settings = {..., function_padding : '1, 0',...}`
we cast then into a tuple ('1', '0') so they get as iterables and then we typecast them into integers.